### PR TITLE
Setlist Notation Guideline Draft

### PR DESCRIPTION
**Pull request by WBTGS95**

### CODE REVIEW

File: blog/_drafts/Setlist Notation Guideline Draft [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Other jam bands may distinguish between different types of jams depending on how 'deep' or far they get from a given song as written, but we are of the opinion that King Gizz only jam 'inside' existing songs (so far). They occasionally use this process to 'write' their songs in studio (Such as for IDPLMAL](https://kglw.net/releases/ice-death-planets-lungs-mushrooms-and-lava/), then go back and re-learn what they did later, so no doubting they will take the next steps sometime in the future.
+
+Example
+
+##Segue
+A segue is the connection of two different songs. Much like King Gizz often do in their studio albums this is also done live to string multiple songs together into medleys. Totally separate songs (where the music stops and there is a noticeable gap before the start of the next), are designated by a comma within a setlist. Segues have two types designated by ">" or "->":  
+###>  
+This is a soft segue, typically the previous song is completed and the next started as written, but the music never truly stops in between, connected perhaps with a consistent beat or sustained chord. Two separate songs but with a consistent flow through the setlist.
+###_>
```
Comment (alxndr) [inline blog/_drafts/Setlist Notation Guideline Draft:31]: should be `->` not `_>`

**Review by alxndr — COMMENTED** [review]
What should the URL for this post be? That will be part of the filename we'll save it as in the `blog/_posts/` subfolder.

**Comment by WBTGS95** [comment]
Well our first meta post is just /inaugural so I guess this could be /notation-guideline ?

File: blog/_drafts/Setlist Notation Guideline Draft [hunk 2]
```diff
@@ -30,8 +30,11 @@ A segue is the connection of two different songs. Much like King Gizz often do i
...
-This style of playing is new to the band, who are clearly learning more every time they step on stage or into the studio; and it is on us to grow with them and learn how to effectively express ourselves.
+This way of listening to the band relies on dedicated live show tapers who freely provide us all with this audio at significant cost and investment of time, for the love of the culture and the band. From a legal/copyright perspective it all operates in a grey-area based on specific allowances from each venue, and a general blessing from the band. This only works as long as we are considerate and respectful of the work and content produced by others, who at the very least always deserve credit for their effort and may each have a preffered way their work is shared. We're all as weird as each other here, so if in doubt, best to reach out and ensure everything stays above board and friendly.
```
Comment (alxndr) [inline blog/_drafts/Setlist Notation Guideline Draft:39]: "preferred" ??

File: blog/_drafts/Setlist Notation Guideline Draft [hunk 3]
```diff
@@ -30,8 +30,11 @@ A segue is the connection of two different songs. Much like King Gizz often do i
...
+This style of playing is new to the band, who are clearly learning more every time they step on stage or into the studio; and it is on us to grow with them and learn how to best enjoy their music.
```
Comment (alxndr) [inline blog/_drafts/Setlist Notation Guideline Draft:40]: imho this last sentence isn't needed

**Comment by alxndr** [comment]
We'll want to rename and move this file within the branch, to `/setlist/_posts/YYYY-MM-DD-notation-guideline.md` and then commit & push that to the PR. Then we'll be able to click "merge" on the PR and it will publish normally.

**Pull request merged**